gce-scripts/packages_to_install.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Installs all the necessary packages for pshtt to run.
+# Logs which package it is installing as well as it's success (0) or failure
+# (1).
+echo 'UPDATE'
+apt-get -y update -qq
+echo $? ' ERROR CODE'
+echo 'PYTHON PIP'
+apt-get -y install python-pip -qq
```

Comment:
Is `python-pip` required in addition to `python3-pip` below? I don't think we have any Python 2 dependencies anymore.